### DISCUSSION TITLE
FDS6.7.7_SMV6.7.18 software dowload for MacOS

### DISCUSSION THREAD

**Discussion by akshithmandala** [firemodels/fds#12957] [General]
Hi...
I am unable to install FDS6.7.7_SMV6.7.18 on my MacOS.
<img width="570" alt="Screenshot 2024-05-18 at 10 07 11 AM" src="https://github.com/firemodels/fds/assets/137080795/e2db2e00-39f3-4727-b037-afb91727aeb1">
I have reached till here... What I do next to get it installed in my mac?****

**Comment by TristanHehnen** [comment]
Apparently, the installation itself has worked. You are only left to configure the path and reload the terminal.

Under point 1 it tells you that you should open the `.bashrc` file. This file should be located in your home directory, likely there: `/Users/YourUserName/ .bashrc`. The following two lines need to be added:
```
source /Applications/FDS/FDS6/bin/FDS6VARS.sh
source /Applications/FDS/FDS6/bin/SMV6VARS.sh
```
Navigate to your home directory, open the `.bashrc` with a plain text editor and add these two lines. You could just put them to the end of the file. 
Remember: files with a period in front are [hidden files in Unix systems](https://askubuntu.com/questions/1072377/what-does-the-period-mean-at-the-start-of-a-filename-in-a-terminal-command). 


Once these are added, you need to restart your terminal (or source the `.bashrc` again). 
That would be point 2.

**Comment by TristanHehnen** [comment]
Also: The title of this discussion should rather be "software installation on MacOS", since this is what you are stuck with - not the download.

**Comment by akshithmandala** [comment]
Thank you for the reply sir. I am unable to find the .bashrc file in my
directory. Could you help me with that?

On Sat, May 18, 2024 at 12:44 PM FireTristan ***@***.***>
wrote:

> Also: The title of this discussion should rather be "software installation
> on MacOS", since this is what you are stuck with - not the download.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/firemodels/fds/discussions/12957#discussioncomment-9477201>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BAV27WYV66LJC5DHS2ZOTPLZC35VBAVCNFSM6AAAAABH5EMQ56VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TINZXGIYDC>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Akshith Goud*
*3rd Year Undergraduate *
*Indian Institute of Technology, Gandhinagar.*

**Reply by TristanHehnen** [reply]
In the terminal, does the file show up when you are in your home directory and type `ls -a`?

**Comment by TristanHehnen** [comment]
Sorry, I just realised that you use `zsh` and not `bash` from your screenshot. The file you are looking for is then the `.zshrc` instead. 

Open the `.zshrc` and add the following lines ([from the Installation Instructions](https://github.com/firemodels/fds/wiki/Installing-and-Running-FDS-under-macOS)):
```
source /Applications/FDS/FDS6/bin/FDS6VARS.sh 
source /Applications/FDS/FDS6/bin/SMV6VARS.sh

#  set OMP_NUM_THREADS to max of 4 and "Total Number of Cores" obtained from running
#  system_profiler SPHardwareDataType
export OMP_NUM_THREADS=4

# unlimit stack for macOS
ulimit -s 65532
```
